designs/virtualization/4.5/Scheduling/Affinity/Affinity.md

```diff
@@ -0,0 +1,65 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Affinity
+
+Introducing Affinity as it appears in-page and the Edit modal.
+
+## in-page placement
+
+![Affinity field in the Scheduling section ](img/Affinity-0-0.jpg)
+The affinity field is placed in the Workload details page > Details tab > Scheduling section > Policies columns.
+
+![Affinity field in the Scheduling section ](img/Affinity-0-1.jpg)
+The affinity field counts the number of rules applied.
+
+## Affinity rules list
+
+![Affinity all rules modal ](img/Affinity-1-0.jpg)
+On clicking on the Edit pencil icon, the user will be presented with a modal, containing all the applied rules for this workload.
+Clicking on the 'Add affinity rule' button will replace this modal with the 'Add affinity rule' modal.
+Selecting 'Edit' from each existing rule kebab menu will replace this modal with the 'Edit affinity rule' modal
+
+A support text below the header will notify the user that:
+"Rules with ‘Preferred’ condition will stack with an ‘AND’ relation between them. 
+Rules with ‘Required condition will have an ‘OR’ relation between them."
+
+The "learn more about affinity rules" links to:
+https://kubevirt.io/2020/Advanced-scheduling-with-affinity-rules.html
+
+## Add/Edit affinity rule modal
+
+![Affinity Edit/Create modal ](img/Affinity-2-0.jpg)
+The default fields for the following fields are:
+- type: Node affinity
+- Condition: Preferredd
+- Weight: 1 (out of 100)
+- Expression type: Label
+- Operator (between Key and Value): In
+
+There is no need to mention the Default to the user.
+
+![Affinity Edit/Create modal - Required ](img/Affinity-2-1.jpg)
```

Comment:
There are inconsistencies with what's been implemented versus designed. That's pretty normal, but we should update these design to reflect what's in the UI to avoid confusion
<img width="807" alt="Screen Shot 2020-05-08 at 11 47 03 AM" src="https://user-images.githubusercontent.com/287880/81417619-d57e6480-9121-11ea-8cae-a1e960514868.png">